### DISCUSSION TITLE
Compatibility issue with Phython

### DISCUSSION THREAD

**Discussion by explorepar** [splunk/slack-alerts#39] [Q&A]
<img width="784" alt="Slack add on phython issue" src="https://user-images.githubusercontent.com/47380225/228686967-7db79a1d-ec04-451c-9c86-3d75ae99e409.PNG">

Splunk Upgrade readiness App is flagging this app. This app is not compatible with Phython 3. The version of app we are running is 2.3.0

How to fix it.

**Comment by ryanwoconnor** [comment]
This would involve upgrading the app to Python 3. It's not really a "fix" as much as an overhaul of the application itself. It's a valid request, however because this is an open source project (and not an official Splunk Product) it may take some time to get in place.
